### PR TITLE
Added Brain Stone Mod

### PR DESCRIPTION
**Pull request by BrainStone**

### CODE REVIEW

File: NEMP/mods.json [hunk 1]
```diff
@@ -4561,5 +4561,14 @@
             "promo": "recommended",
             "promoType": "version"
         }
+    },
+    "Brain Stone Mod": {
```
Comment (Pyker) [inline NEMP/mods.json]: This needs to be the exact name your mod has in Not Enough Mods (`BrainStoneMod`).

File: NEMP/mods.json [hunk 2]
```diff
@@ -4561,5 +4561,14 @@
...
+        "function" : "CheckCurse",
+        "active" : true,
+        "curse" : {
+            "id": "250836",
+            "name": "brainstonemod",
```
Comment (Pyker) [inline NEMP/mods.json]: This is the slug part from the [Curse project page URL](https://mods.curse.com/mc-mods/minecraft/250836-brain-stone-mod), so it should be changed to `brain-stone-mod`. The ID portion is correct.

**Review by Pyker — CHANGES_REQUESTED** [review]
Welcome! I left a few comments on some things that need to be fixed before the pull request is merged.

**Comment by BrainStone** [comment]
I fixed your requests.

**Review by Pyker — APPROVED** [review]
Looks good now, thank you :+1:

**Pull request merged**